app/src/main/res/layout/merge_pretty_empty_view.xml

```diff
@@ -0,0 +1,26 @@
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/empty_view"
+  android:layout_width="wrap_content"
+  android:layout_height="wrap_content"
+  android:layout_gravity="center"
+  android:gravity="center_horizontal"
+  android:orientation="vertical"
+  tools:parentTag="LinearLayout">
+
+  <ImageView
+
+    android:id="@+id/illustration"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:tint="?colorAccent"
+    android:src="@drawable/illustration_lunch"
+    android:contentDescription="Sad picture of a panda" />
```

Comment:
TBH if it's not important for a11y we should simply not set one (@ataulm might confirm here)